### PR TITLE
Support param exec_mem_limit for spark-doris-connctor

### PR DESCRIPTION
**Pull request by Youngwb**
For #2774

### CODE REVIEW

**Review by kangkaisen — DISMISSED** [review]
+1. LGTM.

File: extension/spark-doris-connector/src/main/thrift/doris/DorisExternalService.thrift [hunk 1]
```diff
@@ -56,6 +56,8 @@ struct TScanOpenParams {
   11: optional i16 keep_alive_min
 
   12: optional i32 query_timeout
+
+  13: optional i64 mem_limit
```
Comment (morningman) [inline extension/spark-doris-connector/src/main/thrift/doris/DorisExternalService.thrift]: Plz add comment for this field. Or to change it to a more explicit name, eg: `query_mem_limit`

Comment (Youngwb) [reply extension/spark-doris-connector/src/main/thrift/doris/DorisExternalService.thrift]: Done

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**